master/doc/munin.conf.pod.in

```diff
@@ -204,6 +204,21 @@ Unfortunately, due to the limited functionality of the SSL module in use, it is
 not possible to provide finer-grained filtering.  By default this value is not
 defined.  Affects: munin-update.
 
+=item B<process_timeout_update> <value>
+
+This directive will set the maximum amount of time the munin-update task may run.
+Default value is 240 so we'll make sure the update ended within the 5 minutes
+timespan needed to have complete graphs without gaps.
+
+Do not set this value higher, unless all addresses are using ssh with munin-async.
```

Comment:
Probably *nodes* instead of *addresses* would be better here?

Why would *munin-async* be helpful, here? Exceeding the 5-minutes period would still lead to the next run being skipped due to the log, or?

Regarding the name: I assume, the setting limits the time used for the transmission of data from all nodes? Maybe you can find a better name for this (avoiding the generic *process* term and the probably misleading *update* term (which would include graph processing and limit handling)).

Reply:
I changed the config options and added some more explanation to this option. Is it more clear now, or shall I need to some extra iteration?

Reply:
It looks good to me now. Thanks!